pkg/convertor.go

```diff
@@ -35,7 +46,11 @@ func computeResourceID(resource parser.TerraformResource) string {
 	v := func(name string) string {
 		return fmt.Sprint(resource.AttributeValues[name])
 	}
+
+	var condition map[string]any
```

Comment:
Given this variable is used only in a smaller subset of resources can this be inlined using `:=` and removed from here?